src/main/java/org/kuali/kfs/fp/document/web/struts/DisbursementVoucherAction.java

```diff
@@ -57,6 +57,7 @@
 import org.kuali.kfs.sys.businessobject.PaymentMethod;
 import org.kuali.kfs.sys.businessobject.WireCharge;

```

Comment:
In addition to removing a few imports (such as this one), it looks like the upgrade removed the `retrieveWireChargeMessage()` method entirely. Please include that change in this file.

Reply:
With this next set of changes, I will be trying to make this customized overlay file more closely align to the corresponding base code file.  There are a number of places where "final" is missing from our version as well as "this" existing in our version.  Question: Do you want me to make those items reflect base code OR should I mark those items as a CU Customization?  IMHO one or the other should be done to make future upgrade compares easier.

Reply:
With regards to the various minor differences (like the usage of "final" or "this"), they should NOT be marked as customizations; it would be best to make them line up with base code. If you think such rework expands the scope of the upgrade changes too much, then I would suggest creating a follow-up user story (or multiple such stories) to perform this cleanup. Thanks for checking.

Reply:
The latest set of changes for this file more closely aligns it to base code so that our customization is obvious when performing future upgrades.